libs/common/include/launchdarkly/bindings/c/config/client_side/builder.h

```diff
@@ -0,0 +1,486 @@
+/** @file */
```

Comment:
This is almost the same file as before, except all the Custom/Basic logging builders have been extracted out into logging_builder.h. The intention is to be able to include that in the server-side config builder.